src/test/java/hudson/plugins/jira/JiraCreateIssueNotifierTest.java

```diff
@@ -0,0 +1,176 @@
+package hudson.plugins.jira;
+
+import com.atlassian.jira.rest.client.api.domain.Component;
+import com.atlassian.jira.rest.client.api.domain.Issue;
+import com.atlassian.jira.rest.client.api.domain.Status;
+import hudson.EnvVars;
+import hudson.Launcher;
+import hudson.model.*;
```

Comment:
There is no difference between test and production code, so reasons are the same:
  * cluttering local namespace
  * with a few wildcard imports became absolutely unclear where the given types coming from

It's a matter of taste actually, but... if you don't use wildcard imports nobody will ask you to replace them with wildcard, right? :)